src/configure/configure.ts

```diff
@@ -493,6 +499,10 @@ class Orchestrator {
         }
         let pipelineMap = this.getMapOfUniqueLabels(appropriatePipelines);
         let pipelineLabels = Array.from(pipelineMap.keys());
+        if (pipelineLabels.length === 0) {
+            telemetryHelper.setTelemetry(TelemetryKeys.UnsupportedLanguage, Messages.languageNotSupported);
+            throw new Error(Messages.languageNotSupported);
+        }
```

Comment:
https://mseng.visualstudio.com/AzureDevOps/_workitems/edit/1733189/

Reply:
Rather than failing here, we should use the simple application template.

Reply:
and that should happen for both azure pipelines and github worklow

Reply:
This helps us in handling the existing bug. For scenario, where repo contains php and dockerFile, it will be an issue as we are supporting docker language but not having a template in azurePipeline. So, keeping this should be a good as it will handle the existing bug.